addons/website_sale/views/product_views.xml

```diff
@@ -173,6 +173,16 @@
         </field>
     </record>
 
+    <record id="product_ribbon_view_tree" model="ir.ui.view">
+        <field name="name">product.ribbon.tree</field>
+        <field name="model">product.ribbon</field>
+        <field name="arch" type="xml">
+            <tree string="Products Ribbon">
+                <field name="html" string="Name"/>
```

Comment:
Does this not need an update to the `.pot` file?

Reply:
You are right, completely forgot about it ! Just fixed it

Reply:
Actually the source term is already existing, so you just added a comment on the .pot file :)